CoffeeChat/SceneDelegate.swift

```diff
@@ -45,16 +47,16 @@ class SceneDelegate: UIResponder, UIWindowSceneDelegate {
                         UserDefaults.standard.set(userSession.refreshToken, forKey: Constants.UserDefaults.refreshToken)
                         UserDefaults.standard.set(userSession.sessionExpiration, forKey: Constants.UserDefaults.sessionExpiration)
                         navigationController.pushViewController(rootVC, animated: false)
+                        window.rootViewController = navigationController
+                        self.window = window
+                        window.makeKeyAndVisible()
```

Comment:
Careful here! This will not set any key window if no user is signed in and if `refreshUserSession` has not yet returned. The user will probably get a black screen until you set a key+visible window. You should at the very least show a splash screen. I would move these three lines maybe to line 23; that way you guarantee they execute, and the user isn't presented with a black screen.